src/main/java/frc/robot/Subsystems/ElevatorSubsystem.java

```diff
@@ -151,4 +180,5 @@ public void zeroEncoders() {
     public RelativeEncoder getDrivingEncoder() {
```

Comment:
nit: unless we need to operate on the encoder directly, we should simplify this to just return the position of the encoder.